src/axis.ts

```diff
@@ -23,7 +23,7 @@ import { Pareto} from "./pareto";
   
     g.append("g")
         .attr("transform", "translate(0," + (valueInPixels.height - 50) + ")")
-        .call(d3.axisBottom(categoryAxis).scale(categoryAxis).tickSize(0).tickPadding(3))
```

Comment:
Why was ticksize and tickpadding removed?

Reply:
I removed that to be able to show the ticks on the category axis again